src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo | null;
```

Comment:
```suggestion
  todo: Todo;
```
You don't need to render modal without todo